### ISSUE TITLE
Uncaught TypeError: Cannot read property 'bubbles' of undefined

### ISSUE THREAD

**Issue by rvlira** [sciactive/pnotify#154]
Hi! I'm trying out PNotifyand ran into the error specified in the title: <b>Uncaught TypeError: Cannot read property 'bubbles' of undefined</b>.

This happens when I have an active Desktop Notification on Chrome (37.0.2062.124) and click on it. Because of that exception my code stops working.

The error occurs in line 1742 (PNotify 2.0.1, downloaded from MainSite, all checkboxes marked), when it tries to call <b>event_object.initMouseEvent(e, orig_e.bubbles, ........</b>.

If I click on the "X" button of the notification, my function is able to open the notification again after a while (until the user take some action). But if the user click anywhere else the exception is thrown and my code stops working.

This is what I'm using to create the notification (there are many parameters that I know it's not needed for a desktop notification, but I'm using them just in case it falls back to the regular notification inside my page):

``` js
var myNotification;
myNotification = new PNotify({
                    title: 'Information',
                    text: 'You need to take action.',
                    hide: false,
                    type: 'info',
                    nonblock: {
                        nonblock: true,
                        nonblock_opacity: .2
                    },
                    desktop: {
                        desktop: true
                    },
                    after_close: function (PNotify, timer_hide) {
                        //I added this code so I can clear my variable and allowing my code to reopen
                        //the notification by checking the variable value. Without this my code opens
                        //multiple notifications with the same message, which is what I'm trying to avoid here.
                        //I only want to open/reopen if the notification is closed.
                        myNotification = null;
                    }
                });
```

Edit: I tried not having the "after_close" on my code but the error still happens when I click the notification.

Basically, I want a sticky desktop notification (that will still be sticky if the user's browser do not support it). To avoid opening the same notification over and over I'm checking against the variable holding the notification reference.

Could anyone help me making this code work?

Thanks in advance.

**Comment by buddalasunil999** [comment]
I am facing the same issue. What's the solution to this problem?

**Comment by buddalasunil999** [comment]
solved it. I was using wrong values for stack. The error occurs at the below code. Corrected the stack directions and it doesn't throw any error.

// Calculate the current pos1 value.
                    var csspos1;
                    switch (stack.dir1) {
                        case "down":
                            csspos1 = "top";
                            break;
                        case "up":
                            csspos1 = "bottom";
                            break;
                        case "left":
                            csspos1 = "right";
                            break;
                        case "right":
                            csspos1 = "left";
                            break;
                    }
                    curpos1 = parseInt(elem.css(csspos1).replace(/(?:\..*|[^0-9.])/g, ''));


Also I had to refer this guy to solve the other problem with stacking.
https://github.com/sciactive/pnotify/issues/23

**Comment by timo002** [comment]
This error still exist.

I noticed that when I put the code inside jquery `document ready`, it fails when I click on the notification. But when I put is outside `document ready`, it works perfect.
